c10/core/WrapDimMinimal.h

```diff
@@ -12,11 +12,11 @@ namespace detail {
 // This template can only be specialized at int64_t and c10::SymInt;
 // you'll get linker errors otherwise
 template <typename T>
-C10_API T maybe_wrap_dim_slow(T dim, T dim_post_expr, bool wrap_scalar);
+C10_API T maybe_wrap_dim_slow(T dim, const T& dim_post_expr, bool wrap_scalar);
 } // namespace detail
 
 template <typename T>
-T _maybe_wrap_dim(T dim, T dim_post_expr, bool wrap_scalar = true) {
+T _maybe_wrap_dim(T dim, const T& dim_post_expr, bool wrap_scalar = true) {
```

Comment:
Still not sure about making this const ref

Reply:
We can discard this change in the PR.

Reply:
Reverted